packages/sdk-cli/src/api/developerRelay.ts

```diff
@@ -54,6 +66,13 @@ export async function connect(hostID: string) {
 
 export async function hosts() {
   const response = await apiFetch('1/user/-/developer-relay/hosts.json')
+    .then(async (response: Response) => {
```

Comment:
All of the APIs could respond 403. Why only handle it nicely in `hosts`? The improved 403 error reporting could be added to`assertOK`.

Reply:
Are we guaranteed to have the same schema for error responses on other API endpoints? If so we can do this, but I figured for now we know the schema for the hosts 403 response so handle it in this one case.

Reply:
I also had the thought to make a more generic error handler where you could pass in object with status codes mapped to expected response types, but it didn't seem worth it for this particular change, maybe it is?

Reply:
> Are we guaranteed to have the same schema for error responses on other API endpoints? 
Yes we are. I drove Liam crazy while making sure that was the case.

> I also had the thought to make a more generic error handler where you could pass in object with status codes mapped to expected response types
I was thinking about something along those lines too. It may be worthwhile to do as part of a more holistic change to improve API response error handling across the board, though it probably is overkill for now.